frontend/lib/swr-cache/index.ts

```diff
@@ -15,8 +15,9 @@
  *   e.g. `const productList = await ProductList.get({ ... })`
  */
 
-import { APP_ORIGIN } from '@config/env';
+import { APP_ORIGIN, VERCEL_URL } from '@config/env';
 import { log as defaultLog, Logger } from '@ifixit/helpers';
+import { request } from 'https';
```

Comment:
Does this work on localhost? We're not using `https` during localhost development, so I think this should break 🤔

Reply:
I haven't tried testing on localhost, so it's very possible that this does not work as intended on localhost. Looking at it again, we can probably use the `http` module and specify the `protocol` (http or http) based on the presence of the VERCEL_URL.

Reply:
I think of have to use both, and pick the right one based on the env

Reply:
I think the modules need to be switched based on the url. We can either do an in code switch or use a 3rd party http module which provides a combined interface.

```
import http from "node:http";
import https from "node:https";
const httpModule = url.startsWith("https://") ? https : http;
```